doc/build.txt

```diff
@@ -18,6 +19,9 @@ all/the specified platforms.
     --emulator ......................... Deploy to an emulator
     --target ........................... Deploy to a specific target
 
+    --buildConfig....................... Use the specified build configuration
```

Comment:
:+1: ... At the very least, it should have (on line 9) `--buildConfig=CONFIGFILE` or something like that.